grid-client/deployer/tf_plugin_client.go

```diff
@@ -346,3 +362,46 @@ func (t *TFPluginClient) BatchCancelContract(contracts []uint64) error {
 func generateSessionID() string {
 	return fmt.Sprintf("tf-%d", os.Getpid())
 }
+
+// isTwinVerified makes sure the twin used is verified
+func isTwinVerified(twinID uint32, net string) (verified bool) {
```

Comment:
why not returning the errors? then we can log them if not needed when using the function